tests/test_generators.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+
+from transformers import pipeline
+
+from adatest import generators
+
+
+class TestTransformers:
+    def test_smoke(self):
+        hf_model = pipeline("text-generation", model="EleutherAI/gpt-neo-125M")
+        target = generators.Transformers(hf_model.model, hf_model.tokenizer)
+
+        prompts = [
+            ("id A", "Great hotel", "positive"),
```

Comment:
I'm not sure that this is how the data are supposed to be presented. This at least gets me a result, but it's not exactly what I was expecting.